lib/daysOfTheWeek.js

```diff
@@ -9,3 +9,16 @@ module.exports.getDayName = (date) => {
 
   return ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'][passedDate.getDay()];
 };
+
+module.exports.isWeekend = (date) => {
```

Comment:
This code adds a new function but doesn't add any new tests.  This new function is completely untested!